globalSetup.js

```diff
@@ -0,0 +1,6 @@
+// Need to import 'regenerator-runtime/runtime' here instead of in
+// jest.setup.js, otherwise, import 'recurly.js/test/server' fails because
+// it uses async/await.
+require('regenerator-runtime/runtime');
+require('recurly.js/test/server');
+export default async function () {}
```

Comment:
Importing the test server here runs it, but jest still expects an async function to be exported from this file https://jestjs.io/docs/en/configuration#globalsetup-string